keyserver/handler.go

```diff
@@ -56,3 +65,12 @@ func (server *ConiksServer) handleRegistrationMessage(reg *p.RegistrationRequest
 		TB:   tb,
 	}, nil
 }
+
+func (server *ConiksServer) StoreUserPoliciesToKV(up *p.ConiksUserPolicies) error {
```

Comment:
> I'm kind of confused here. Why do we need to store the user policies to both db and the data blob mapped to the user's index? 

Sorry. I probably didn't explain clearly what I meant. What I'm trying to say is that if the data blob is already in the DB, and we store the user's policies as part of the blob, we don't need to also store them in the DB again. The current implementation in ``handler.go`` seems to be storing the user policies in a separate entry in the DB, and the main goal of my suggestion was to save space in the DB and store all user-related data in the same place, i.e. the blob. I hope this makes more sense.

> Another question, do we want to preserve the user's privacy (in this case, it is the user policies) from the lookup operation? Could we just return all the data blob to the user and the user policies would be exposed; or do we have to do some kind of filtering/trimming the data blob before returning to the client?

Good question. I think the policies must be revealed to the client, otherwise the client won't be able to parse the rest of the data blob. In a proof of absence, the policies shouldn't be included anyway since the entire blob is dropped from the proof. Bu if the server removes the user policies before responding to a proof of inclusion, how will the client know if the data it's receiving is the user's key or an encrypted blob? And how will the client know if it should look for a signature on a key change message? 

In the case of the lookup policy, revealing the policy to the client shouldn't violate the user's privacy if the client isn't authorized to decrypt the encrypted blob. In the case of the key change policy, I do think there might be a risk to exposing the policy to a malicious client (which might attempt to change a user's key if it detects that the user allows unsigned key changes). But I think the solution to this issue might be some kind of account authentication again.